test/metabase/task/cleanup_temorary_computation_job_results_test.clj

```diff
@@ -0,0 +1,27 @@
+(ns metabase.task.cleanup-temorary-computation-job-results-test
+  (:require [clj-time
+             [coerce :as t.coerce]
+             [core :as t]]
+            [expectations :refer :all]
+            [metabase.models
+             [computation-job :refer [ComputationJob]]
+             [computation-job-result :refer [ComputationJobResult]]]
+            [metabase.task.cleanup-temporary-computation-job-results :as task]
+            [toucan.db :as db]
+            [toucan.util.test :as tt]))
+
+(tt/expect-with-temp
+ [ComputationJob       [{job-id :id} {:type   :simple-job
+                                      :status :done}]
+  ComputationJobResult [{result-id :id} {:job_id     job-id
+                                         :permanence :temporary
+                                         :payload    1}]]
+  true
+  (let [c1 (count (ComputationJobResult))]
```

Comment:
you can do `(db/count ComputationJobResult)` instead. That produces a SQL query like `SELECT COUNT(*) ...` whereas doing `count` in Clojure-land pulls down every object from the DB with `SELECT *` and *then* counts them.